commands.js

```diff
@@ -1,13 +1,28 @@
 'use strict';
 const fs = require('fs');
 const chalk = require('chalk');
+const pathFile = 'input.json';
+
+function ifExist(path) {
```

Comment:
I would name it something like `createIfNotExists` or `ensureExists`. From `ifExists` it is not fully clear what it does.